lua/telescope/pickers.lua

```diff
@@ -783,6 +795,7 @@ function Picker:set_selection(row)
   end
 
   state.set_global_key("selected_entry", entry)
+  state.set_global_key("original_opts", self.original_opts)
```

Comment:
It worries me keeping around the original opts because we may have weird cases of leaving memory cycles & therefor not garbage collecting stuff. Not sure if it will happen though.

Reply:
https://github.com/nvim-telescope/telescope.nvim/pull/290#discussion_r534726522 kinda was your idea keeping them 😆

Reply:
But thinking about it now i think i don't need to keep the opts. I will try something the next time i work on it :)